src/breakpoints/index.ts

```diff
@@ -130,13 +111,32 @@ export namespace Breakpoints {
       Signal.clearData(this);
     }
 
-    private _selectedType: SessionTypes;
+    setBreakpoints(id: string, breakpoints: IBreakpoint[]) {
+      this._breakpoints.set(id, breakpoints);
+      this.changed.emit(breakpoints);
+    }
+
+    getBreakpoints(id: string): IBreakpoint[] {
+      return this._breakpoints.get(id) || [];
+    }
+
+    restoreBreakpoints(breakpoints: Map<string, IBreakpoint[]>) {
+      this._breakpoints = breakpoints;
```

Comment:
So it looks like the main difference between `setBreakpoints` and `restoreBreakpoints` is that `restoreBreakpoints` sets all breakpoints at once, while `setBreakpoints` will do it id by id.

Would it make sense to also use the `changed` signal when restoring the breakpoints, or do we expect the restore logic to be different later and require an additional `restored` signal?

Reply:
I'm not sure yet (see my reply to your comment about the `restore`/`update` functions in `body.tsx`

Reply:
:+1: